packages/plugins/model-evaluate/image-generation-tensorflow-model-evaluate/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { ModelEvaluateType, UniModel, ImageDataset, EvaluateResult, ArgsType } from '@pipcook/pipcook-core';
 
 const cycleGanModelEvaluate: ModelEvaluateType
-  = async (data: ImageDataset, model: UniModel, args: ArgsType): Promise<EvaluateResult> => {
+  = async (data: ImageDataset, model: UniModel, _args: ArgsType): Promise<EvaluateResult> => {
```

Comment:
We should support the `ModelEvaluateType` could be compatible with `(data: ImageDataset, model: UniModel) => Promise<EvaluateResult>` instead of marking this to be ignored by lint tools.